src/test/java/com/wikia/webdriver/common/templates/TemplateNoFirstLoad.java

```diff
@@ -11,12 +12,19 @@
   protected void loadFirstPage() {
   }
 
-  public AdsBaseObject openPageWithVideoInLocalStorage(Page page) {
-    final AdsBaseObject ads = new AdsBaseObject(driver);
-    ads.getUrl(urlBuilder.getUrlForWiki("project43"));
-    JavascriptActions runScript = new JavascriptActions(driver);
-    runScript.execute("localStorage.setItem('" + VuapVideos.PORVATA_VAST + VuapVideos.VAST_VIDEO + ");");
+  protected AdsBaseObject openPageWithVideoInLocalStorage(Page page, VideoAd videoAd) {
+    final AdsBaseObject ads = new AdsBaseObject(driver, urlBuilder.getUrlForWiki("project43"));
+    putVASTToLocalStorage(videoAd.getVastXML());
     ads.getUrl(page);
     return ads;
   }
+
+  protected AdsBaseObject openPageWithVideoInLocalStorage(Page page) {
+    return openPageWithVideoInLocalStorage(page, VuapVideos.DEFAULT);
+  }
+
+  private void putVASTToLocalStorage(String vast) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "putVASTToLocalStorage" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)